NetKAN/AntennaRangePatches.netkan

```diff
@@ -0,0 +1,35 @@
+{
+	"spec_version"	: "v1.4",
+	"identifier"	: "AntennaRangePatches",
+	"$kref"		: "#/ckan/kerbalstuff/1220",
+	"$vref"		: "#/ckan/ksp-avc",
+	"license": "CC-BY-NC-SA-4.0",
+	"resources"	:
+	{
+		"kerbalstuff" : "https://kerbalstuff.com/mod/1220",
+		"repository"  : "https://github.com/linuxgurugamer/AntennaRangePatches"
+	},
+	"depends": [
+		{ "name": "AntennaRange" },
+		{ "name": "ModuleManager" }
+	],
+	"conflicts": [
+		{ "name": "AntennaPatch4AntennaRange"},
+		{ "name": "RemoteTech"}
+	],
+	"install": [
+		{
+			"find"		: "AntennaRangePatches",
+			"install_to"	: "GameData"
+		},
+		{
+			"find"		: "AutoPruner",
+			"install_to"	: "GameRoot"
+		}
+	],
+	"recommends": [
+		{ "name": "RemoteTech" },

```

Comment:
ok.

I goofed, sorry

now I'm confused.

is this already fixed and committed?

Before I do another pull request?  i don't see it there.

I've removed the AutoPruner and removed the recommends RT line, so that 
should be ok.

Thanks

JBB

On 10/27/2015 8:40 AM, Matthew wrote:
>
> In NetKAN/AntennaRangePatches.netkan 
> <https://github.com/KSP-CKAN/NetKAN/pull/2490#discussion_r43115560>:
>
> > +	"conflicts": [
> > +		{ "name": "AntennaPatch4AntennaRange"},
> > +		{ "name": "RemoteTech"}
> > +	],
> > +	"install": [
> > +		{
> > +			"find"		: "AntennaRangePatches",
> > +			"install_to"	: "GameData"
> > +		},
> > +		{
> > +			"find"		: "AutoPruner",
> > +			"install_to"	: "GameRoot"
> > +		}
> > +	],
> > +	"recommends": [
> > +		{ "name": "RemoteTech" },
>
> You cannot recommend a mod you conflict with.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/KSP-CKAN/NetKAN/pull/2490/files#r43115560>.
>